packages/cli/src/type-generator.ts

```diff
@@ -161,6 +161,7 @@ export default class TypeGenerator {
             GENERATED_FILE_NOTE,
             ...codeGenerator.generateModuleImports(),
             ...codeGenerator.generateTypes(),
+            ...codeGenerator.generateDerivedLoaders(),
```

Comment:
we should add changeset explaining the changes what this feature brings in, so we can have it documented in changelog.

Reply:
Added changesets